tools/scripts/acr-build.ps1

```diff
@@ -79,7 +79,13 @@ else {
         # dev feature builds
         $namespace = $namespace.Replace("feature/", "")
     }
-    elseif ($namespace.StartsWith("release/") -or ($namespace -eq "main") -or ($namespace -eq "master")) {
+    elseif (($namespace -eq "main")) {
+        if ([string]::IsNullOrEmpty($Registry)) {
+            # Builds from main should be built nightly and per PR in a container registry to allow testing.
+            $Registry = "industrialiotdev"
+        }
+    }
+    elseif ($namespace.StartsWith("release/") -or ($namespace -eq "master")) {
```

Comment:
Main is same as master.  You should be able to remove the test for master and main from the if statement and get the same results since the else is the same as the newly added code

Reply:
I wanted to leave 'master' related changes untouched. Happy to do it next PR.

Reply:
It is not related to master.  My point was to remove the test for both and you end up with the same result as what the added code provides (the default path uses the branch name as namespace name and builds to industrialiotdev)

Reply:
Good point. I'll refactor this.